core/scripts/Voting.js

```diff
@@ -201,10 +207,11 @@ async function runVoting() {
 }
 
 run = async function(callback) {
-  var request = { identifier: { first: "BTC", second: "USD" }, time: "1560762000" };
+  var testRequest = { dataSource: "CryptoCompare", identifier: { first: "BTC", second: "USD" }, time: "1560762000" };
+  var ccRequest = { dataSource: "test", identifier: "BTCUSD", time: "1560762000" };
```

Comment:
Let's go one step further here and build a map for all supported identifiers. So maybe something like:
```
const SUPPORTED_IDENTIFIERS = {
    "BTCUSD": { dataSource: "CryptoCompare", <whatever else> },
     "test": { dataSource: "test" },
}
```
 
Then, in `fetchPrice`, you can do:
```
const config = SUPPORTED_IDENTIFIERS[request.identifier];
switch (config.dataSource) {
    case "CryptoCompare":
        return await fetchCryptoComparePrice(config.first, config.second);
    case "test":
        return "1.5";
}
```

I think that'll nicely separate the configuration of the identifiers from the price fetching logic.

Let me know if you run into trouble with the tests, or if that doesn't make sense.

Reply:
I think this bit works now. Do you think there's an easier way to generalize for more price feeds?